sdk/servicebus/service-bus/test/perf/track-2/sbBase.spec.ts

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { PerfStressTest, getEnvVar } from "@azure/test-utils-perfstress";
+import { ServiceBusAdministrationClient, ServiceBusClient } from "../../../src";
+
+// Expects the .env file at the same level as the "test" folder
+import * as dotenv from "dotenv";
+dotenv.config();
+
+const connectionString = getEnvVar("SERVICEBUS_CONNECTION_STRING");
+const sbClient = new ServiceBusClient(connectionString);
+
+export abstract class ServiceBusTest<TOptions> extends PerfStressTest<TOptions> {
+  sbClient: ServiceBusClient;
+  sbAdminClient: ServiceBusAdministrationClient;
+  static queueName = `newqueue-${Math.ceil(Math.random() * 1000)}`;
+
+  constructor() {
+    super();
+    this.sbClient = sbClient;
+    this.sbAdminClient = new ServiceBusAdministrationClient(connectionString);
+  }
+
+  public async globalSetup() {
+    await this.sbAdminClient.createQueue(ServiceBusTest.queueName);
+  }
+
+  public async globalCleanup() {
+    await this.sbAdminClient.deleteQueue(ServiceBusTest.queueName);
+    await this.sbClient.close();
```

Comment:
I think `sbClient` should be closed in `cleanup()` instead of `globalCleanup()`, since there is one per test class instance.

Reply:
No, because the same sbClient that is created outside is being reused for all the test class instances,